### PR TITLE
Add a NN Learning Rate setter

### PR DESCRIPTION
**Pull request by arisanguinetti**

### CODE REVIEW

**Comment by shiffman** [comment]
Thank you!

**Comment by laerador** [comment]
May I ask, what is the difference between calling the setter `nn.setLearningRate(0.01)` and using the property directly `nn.learning_rate=0.01`?

**Comment by arisanguinetti** [comment]
@laerador it's the same. For me, it seems better to use a setter than go directly to the property
